core/kubernetes/main/k8s-mini.sh

```diff
@@ -16,9 +19,6 @@ elif [ "$1" == "down" ]; then
     kubectl delete deployment hapi-fhir-server-deployment
     kubectl delete deployment hapi-fhir-mysql-deployment
 elif [ "$1" == "destroy" ]; then
-    # delete host entry on destroy
```

Comment:
Was there a reason for removing this in the destroy? This just allows for cleanup of the host file so that the user doesnt have this entry in there forever

Reply:
This was an accident :|